recipes/openmesh/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+source activate "${CONDA_DEFAULT_ENV}"
+
+if test `uname` = "Darwin"
+then
+  SO_EXT='.dylib'
+else
+  SO_EXT='.so'
+fi
+
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DBUILD_APPS=OFF \
+  -DOPENMESH_PYTHON_VERSION=${PY_VER} \
+  -DOPENMESH_BUILD_PYTHON_UNIT_TESTS=ON \
+  ..
+
+make install -j${CPU_COUNT} VERBOSE=1
+
+# osx, py34: ***Exception: SegFault in test_load_obj_with_material (test_read_write_obj.ReadWriteOBJ)
```

Comment:
I see there is [one test]( https://travis-ci.org/conda-forge/staged-recipes/jobs/152315764#L1789 ) that fails. That is unfortunate. Thanks for sharing. Feel free to add this back.